junitparser/cli.py

```diff
@@ -0,0 +1,50 @@
+from argparse import ArgumentParser
```

Comment:
the whole file is basically the old ` bin/junitparser`
this slight improvements